save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/service/AgentService.kt

```diff
@@ -325,10 +326,10 @@ class AgentService(
             }
             .map { (isStandardMode, testPaths) ->
                 val cliArgs = if (!isStandardMode) {
-                    suitesToArgs.values.first()
+                    FilenameUtils.separatorsToUnix(suitesToArgs.values.first())
```

Comment:
I can see that you are now saving tests in `TestService` with Unix separators. Do we still need conversion here? I believe these values are constructed using existing data from the database

Reply:
`suitesToArgs` is taken from `TestSuiteDto.testRootPath` -- maybe normalize separators on TestSuite level? 
I guess it can be normalize on saving (TestService and TestSuiteService)